packages/buckets/src/buckets.spec.ts

```diff
@@ -145,16 +145,17 @@ describe('Buckets...', () => {
 
     // Recursive dir
     // [
-    //   'mybuck',
-    //   'mybuck/.textileseed',
-    //   'mybuck/dir1',
-    //   'mybuck/dir1/file1.jpg',
-    //   'mybuck/path',
-    //   'mybuck/path/to',
-    //   'mybuck/path/to/file2.jpg'
+    //   '.textileseed',
+    //   'dir1',
+    //   'dir1/file1.jpg',
+    //   'path',
+    //   'path/to',
+    //   'path/to/file2.jpg'
     // ]
```

Comment:
having the flat paths prefixed by the buck name wasn't helpful when building with this list